tools/Newtonsoft.Json.xml

```diff
@@ -0,0 +1,10760 @@
+<?xml version="1.0"?>
```

Comment:
None of these xml files should be included.  You should add them specifically to .gitignore